Resources/translations/sub_items.en.xliff

```diff
@@ -32,18 +37,23 @@
         <note>key: bulk_delete.success.message</note>
       </trans-unit>
       <trans-unit id="364a3498e021163497d9110380736b876beb326f" resname="bulk_move.error.message">
-        <source>You do not have permission to move at least 1 of the selected content item(s). Please contact your Administrator to obtain permissions.</source>
-        <target state="new">You do not have permission to move at least 1 of the selected content item(s). Please contact your Administrator to obtain permissions.</target>
+        <source><![CDATA[%notMovedCount% of the %totalCount% selected item(s) could not be moved due to lack of your user permissions. <u><a class='ez-notification-btn ez-notification-btn--show-modal'>Click here for more information.</a></u><br>Please contact your Administrator to obtain permissions.]]></source>
```

Comment:
```suggestion
        <source><![CDATA[%notMovedCount% of the %totalCount% selected item(s) could not be moved because you do not have proper user permissions. <u><a class='ez-notification-btn ez-notification-btn--show-modal'>Click here for more information.</a></u><br>Please contact your Administrator to obtain permissions.]]></source>
```